src/screens/IndexScreen.js

```diff
@@ -248,6 +250,11 @@ export const IndexScreen = ({ navigation, route }) => {
                     <DropdownHeader {...{ query, queryVariables, data, updateListData }} />
                   ) : null
                 }
+                ListEmptyComponent={
+                  listItems ? (
+                    <EmptyMessage title="Im Moment konnte kein anzuzeigender Inhalt gefunden werden." />
```

Comment:
as mentioned already, we try to avoid hardcoded texts and use references to texts from the file `src/config/texts`.
i could think of something like `texts.empty.list` here.
the text itself could be changes to: "Schade, es wurden keine passenden Einträge gefunden, bitte zu einem späteren Zeitpunkt erneut versuchen."

Reply:
I will update the code in this way.